ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2

```diff
@@ -81,6 +81,13 @@
 {%             else %}
 {%                 set switch.mlag = l3leaf.defaults.mlag | default(true) %}
 {%             endif %}
+{%             if l3leaf.node_groups[l3leaf_node_group].mlag_ibgp_origin_incomplete is defined and l3leaf.node_groups[l3leaf_node_group].mlag_ibgp_origin_incomplete is not none %}
```

Comment:
please refactor to use the new `| arista.avd.default` filter

Reply:
Fixed in commit 49f51f6.